xapian-letor/tests/harness/testutils.h

```diff
@@ -90,4 +90,11 @@ void test_mset_order_equal(const Xapian::MSet &mset1,
 	expected_exception = NULL;\
     } while (0)
 
+/// Check that CODE throws exactly Xapian LetorParseError TYPE.
```

Comment:
This macro doesn't take parameters `CODE` or `TYPE` (unlike `TEST_EXCEPTION()`) so the comment doesn't really make sense.

Also, it's conventional to make macro parameters all-caps.

Reply:
fixed in a fixup commit.